kafka/README.md

```diff
@@ -9,7 +9,7 @@ Kafka is a distributed publish-subscribe messaging system that is designed to be
 
 It makes use of the following technologies:
 
-* [StatefulSets](http://kubernetes.io/docs/concepts/abstractions/controllers/statefulsets/) 
+* [StatefulSets](http://kubernetes.io/docs/concepts/abstractions/controllers/statefulsets/)
 * [Headless Services](http://kubernetes.io/docs/user-guide/services/#headless-services)
 
 ## Prerequisites
```

Comment:
Since applier kind of obfuscates what is actually getting created, can we set an expectation for what will get created when you run the automation? I think most important to point out is that this will attempt to create a project, and the user needs to ensure A) that they can create projects, and B) that the project name isn't taken